libs/platform/value-help-dialog/value-help-dialog/value-help-dialog.component.ts

```diff
@@ -193,7 +195,7 @@ export class PlatformValueHelpDialogComponent<T = any> extends VhdComponent impl
 
     /** Id attribute for dialog title */
     @Input()
-    headerId: string | null = null;
+    headerId = `fd-vhd-header-unique-id-${vhdHeaderUniqueId++}`;
```

Comment:
not a big deal, but in my opinion the prefix should be `dfp-`.
Do we need the word `unique`? Can we shorten the id to `fd-vhd-header-id-${vhdHeaderUniqueId++}`?